utils/ansible_galaxy_meta_template.yml

```diff
@@ -1,5 +1,5 @@
 galaxy_info:
-  author: SCAP Security Guide development team
+  author: Red Hat Security Automation development team
   description: @DESCRIPTION@
 
   issue_tracker_url: https://github.com/OpenSCAP/scap-security-guide/issues
```

Comment:
When you edit this file, could you change this link as well?

Reply:
Oops... missed that one. Fixed @jan-cerny